test/addons/dlopen-ping-pong/test.js

```diff
@@ -14,7 +14,4 @@ process.dlopen(module, bindingPath,
 module.exports.load(`${path.dirname(bindingPath)}/ping.so`);
 assert.strictEqual(module.exports.ping(), 'pong');
 
-// Check that after the addon is loaded with
-// process.dlopen() a require() call fails.
-const re = /^Error: Module did not self-register\.$/;
-assert.throws(() => require(`./build/${common.buildType}/binding`), re);
+assert.doesNotThrow(() => require(`./build/${common.buildType}/binding`));
```

Comment:
Why did this change?

Reply:
i'm assuming because `DLOpen()` doesn't throw on missing self-registration anymore, instead it just abandons

Reply:
@TimothyGu It changed because re-dlopen()-ing a module will just give you the module again, like what you’d expect `require()` to do